mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss

```diff
@@ -1,4 +1,6 @@
 .verticalContainer {
   display: flex;
+  height: 100%;
   flex-direction: column;
+  justify-content: space-between;
```

Comment:
hmm, thats dangerous

we should not change this component

Reply:
Done

Reply:
You added height 100% to the vertical container. But it looks like in project some vertical containers should have different height. 

We should not change this component. Use custom classes to change is behavior locally

Reply:
Done